### PR TITLE
print performance stats between branches

### PR DESCRIPTION
**Pull request by snopoke**
Produces output like this:

```
App                                                         Mem diff %     Time diff %    
----------------------------------------------------------- -------------- -------------- 
project-aa1948da9c6b3a2d0c8b6b9c5e4db99f                    0.07           -16.61         
frhs-who-columbia-8018528db773225435c6c5cd769b9179          0.00           -15.57         
pathfinder-56cc9a9056bcb0c231221c4267102de0                 -0.05          -5.52          
icmr-32592452dbccf21322e7ad0192e3a34f                       -0.02          -30.18         
bahraich-31e8d2ff49f15cc35cccf3bcad6b0e96                   0.00           -29.32         
crs-remind-e3689e9271394d1215f61c11b5d55de1                 0.01           -12.46         
rmf-8065de7253c8ea8bd475d5030b976c47                        0.00           -5.90          
crs.csi-13fb26d1d491f89636a699ddf7fa5cc5                    0.00           3.16           
dtree-familyplanning-d4102331030c2795864e9e1f5e537dcf       0.02           -30.83         
tulasalud-90ea9c9cb34038f64623fb4d082173ad                  0.01           -17.38 
```

### CODE REVIEW

File: corehq/apps/app_manager/management/commands/build_apps.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import contextlib
+from functools import wraps
```
Comment (dimagimon) [inline corehq/apps/app_manager/management/commands/build_apps.py]: F401 'wraps' imported but unused

**Comment by dannyroberts** [comment]
This is a super cool idea—and it looks like the branch you're testing is a lot faster too :)

**Comment by snopoke** [comment]
Thanks, these were the changes I was comparing: https://github.com/dimagi/commcare-hq/pull/5821

Was thinking of trying to get a jenkins build to do this daily and plot a graph but this is as far as I got.

**Comment by czue** [comment]
awesome!

**Pull request merged**